heirloom.postman_collection.json

```diff
@@ -266,35 +266,6 @@
 						}
 					},
 					"response": []
-				},
-				{
-					"name": "upload cover photo",
-					"request": {
-						"method": "POST",
-						"header": [],
-						"body": {
-							"mode": "formdata",
-							"formdata": [
-								{
-									"key": "img",
-									"type": "file",
-									"src": []
-								}
-							]
-						},
-						"url": {
-							"raw": "http://localhost:5000/image",
-							"protocol": "http",
-							"host": [
-								"localhost"
-							],
-							"port": "5000",
-							"path": [
-								"image"
-							]
-						}
-					},
-					"response": []
```

Comment:
it looks like the postman file you uploaded is gonna delete a request. u can make a duplicate and copy it over so it adds the ones you added